### DISCUSSION TITLE
Wooden shovel and planks from a self-reliant perspective

### DISCUSSION THREAD

**Discussion by ghost** [CleverRaven/Cataclysm-DDA#48952] [General]
A wooden shovel requires a plank to be crafted. A plank requires iron tools. Iron tools require a large investment of iron. Once a survivor is able to produce a plank, the need for a wooden shovel disappears as he can just craft an iron shovel instead.

We're able to cut fallen tree trunks into logs using stone tools, but it's not currently possible to employ the same method to cut a log in another axis - to make a relatively flat surfaced piece of wood. In reality, even wildly swigging an axe at wooden logs will produce flat surfaced pieces of wood. It is not far fetched that with enough effort a survivor might be able to flatten both sides of such a wooden piece to serve as a shovel head.

I propose an alternative wooden shovel recipe, one which additionally requires tree cutting quality 1 and instead of a plank it would use a log. Purpose would be to allow crafting of a digging 2 implement using only stone and wood tools, for use in various primitive constructions that currently require digging 2.

Alternatively, ability to craft wooden planks with only primitive tools.

**Comment by anothersimulacrum** [comment]
From previous discussions on the subject, planks aren't a feasible thing to make with primitive tools, and the various things that would be feasible to make from in this scenario should have alternates added to simulate that, instead of planks becoming available.

**Comment by robotic-gamer** [comment]
in theory you could just carve an entire shovel from a log as a single piece of wood. only drawback is it would take a while to craft

**Comment by coyo7e** [comment]
You can craft a plank IRL with a stone adze or other similar wood planing tools but it wastes most of the log, but that's how it used to be done. 

Being able to craft a plank with anything short of a table saw setup or a lot of skills with a chainsaw is not actually very realistic - there's no way to use a hacksaw or bow saw etc to cut down the length of a log, etc.. Even using a circular saw is pretty unrealistic because they can generally only cut about 2-3" deep. Loggers used to use large two-person saws and lift a log up onto braces to saw from above and below along the length, but that's not feasible for a solo person to use a non-bow type of hand saw because it's pretty impossible to make two parallel cuts straight enough to suffice.
